### DISCUSSION TITLE
【spring】[易用性] 面向spring是否应该有简易的命令，比如getbean

### DISCUSSION THREAD

**Discussion by laglangyue** [alibaba/arthas#2463] [Q&A]
当前获取springContext，然后再去调用有一些复杂度，对于无法复制，无法连接外网的环境排查问题太困难，每执行一步，需要敲很长的命令，很容易出错，如果有一些简易的命令，直接通过bean名称获取bean

**Comment by hengyunabc** [comment]
用 vmtool ，看 vmtool wiki。
